lib/course_clone_manager.rb

```diff
@@ -34,6 +29,21 @@ def clone!
 
   private
 
+  def clone_relevant_information
```

Comment:
I don't like this refactor, because not all the steps copied over are really cloning information.

If this is just to satisfy rubocop by not having an extra line, maybe `clear_meeting_days_and_due_dates` could be moved to the end of `duplicate_timeline`, as it's basically part of that step conceptually.

Reply:
Yes this was indeed just to satisfy the rubocop. I will fix this up.